README.md

```diff
@@ -0,0 +1,58 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Restaurant App
+
+It's a Restaurant app to show the restaurant, the menu and to contact the owner information dynamically by clicking in the button in the Nav Bar
+
+![screenshot](./assets/screenshot/screenshot.jpg)
+
+- Has the information about the restaurant
+- Shows the menu.
+- Has a contact section.
+
+## Built With
+
+- Html, CSS, and JavaScript
+- Bootstrap
+
+## Live Demo
+
+[Live Demo Link](https://fdi96.github.io/Restaurant-App/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Install
+
+You can download the code to your personal computer clonig the repository using the following command
+
+```
+git clone
```

Comment:
- [ ] [MANDATORY] It would be better to complete this command to avoid confusion.

Reply:
Done! Thanks for the review.